README.md

```diff
@@ -534,6 +535,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Appetize](https://appetize.io) — Test your Android & iOS apps on this Cloud Based Android Phone/Tablets emulators and iPhone/iPad simulators directly in your browser. Free tier includes 1 concurrent session with 100 minutes usage per month. No limit on app size.
   * [Bird Eats Bug](https://www.birdeatsbug.com/) — Report bugs faster (and better). Record your screen with Bird browser extension, it will auto-capture technical data that engineers need to debug. Free tier suitable for small teams.
   * [BugBug](https://bugbug.io/) - Lightweight test automation tool for web applications. Easy to learn and doesn't require coding. You can run unlimited tests on your own computer for free. For an additional monthly fee, you also get cloud monitoring and CI/CD integration.
+  * [Buildkite Test Analytics](https://buildkite.com/test-analytics) — Get more out of your test suites. Free developer tier includes 100k test executions/month, with more free inclusions for open source projects
```

Comment:
We only list specific companies once to keep the list managable, list this one at the most appropriate section and mention all features you wish to mention

thanks!